tests/dataset_tests/CIFAR-10/Cifar10Test.cpp

```diff
@@ -42,13 +42,13 @@ TEST_F(Cifar10Test, trainNeuralNetwork)
 {
     StraightforwardNeuralNetwork neuralNetwork({
         Input(3072),
-        FullyConnected(100),
-        FullyConnected(25),
-        FullyConnected(10)
+        FullyConnected(80),
+        FullyConnected(30),
+        FullyConnected(10, activation::identity, Softmax())
```

Comment:
Why do you need as activation the identity ?

Reply:
Yes but we don't care. (We're not at work)